sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -163,6 +163,51 @@ final class CastExprMeta[INPUT <: UnaryExpression with TimeZoneAwareExpression w
   override protected val needTimezoneTagging: Boolean = false
 }
 
+trait CastOptions extends Serializable {
```

Comment:
Why have a trait if there is only one implementation of it. If ToPrettyString is going to change that I understand.

Reply:
Yes, it will change it.